### PR TITLE
Mjz/11 offer generator

### PR DESCRIPTION
**Pull request by mzagaja**
Creating this pull request so we can figure out what we need to do in order to appropriately setup the front end for accept/reject.

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
+  resources :offers
```
Comment (allthesignals) [inline config/routes.rb]: Do we not want to namespace this? '/api'

File: config/routes.rb [hunk 2]
```diff
@@ -1,4 +1,5 @@
...
   get 'travel_time/get'
```
Comment (allthesignals) [inline config/routes.rb]: For JSON API, response headers should be `application/vnd.api+json` not `application/json` - don't think this is super urgent (not sure where it should be used in the front end).

**Comment by mzagaja** [comment]
I updated the offers route to namespace it to api. In regards to the response header, is that not done by the format.jsonapi lines in the controller? Let me know what the general approach is and I can update it.

**Comment by allthesignals** [comment]
@mzagaja It's totally fine - I'll just take care of it. Just letting you know. Thanks!

**Pull request merged**